packages/components/src/components/animations/DeviceAnimation.tsx

```diff
@@ -37,7 +37,7 @@ export const DeviceAnimation = forwardRef<HTMLVideoElement, DeviceAnimationProps
             type,
             loop = false,
             shape,
-            // if no Trezor available, show flagship model
+            // default flagship model TODO: change to T3T1
```

Comment:
Maybe we can have a constant called: `DEFAULT_FLAGSHIP_MODEL` somewhere and use it in all places